### ISSUE TITLE
Multiple imputation

### ISSUE THREAD

**Issue by netphar** [sebp/scikit-survival#152]
Hello!

This is not a scikit-survival specific question. But I hope you'd consider answering nevertheless.

I m doing survival analysis using using a number of Python models (NNs, RSF, GB with different base learners) and also using a linear coxPH from survival package in R. Since my data has missing values, I impute them in R using a number of methods and then pool predictions to get imputation-adjusted confidence intervals for linear coxPH baseline(Rubin's rule). `fit.mult.impute` from Hmisc library seems to work pretty well for that.

now I was wondering if you know any library capable of doing that, but for Python models that predict C-index and use loglikelihood as loss? 

thanks!

**Comment by sebp** [comment]
Multiple imputation should be independent on whether the task is survival analysis or something else. [statsmodels](https://www.statsmodels.org/stable/imputation.html?highlight=impute) implements MICE, and sklearn has [something resembling MICE](https://scikit-learn.org/stable/modules/impute.html#multiple-vs-single-imputation). I'm not sure whether either of them implements Rubin's rule, but if you want to pool estimates of linear models, it should be fairly easy to implement yourself.
